### PR TITLE
WD-8434 - update newsletter signup form

### PR DESCRIPTION
**Pull request by lizzochek**
## Done

Update newsletter signup form according to copy doc

## QA
- [demo link](https://ubuntu-com-13545.demos.haus/blog)
- [copy doc](https://docs.google.com/document/d/1VrmfcndrIT8E0K2cLqBpYPNhXECtte5fbLCNy8XcCaw/edit)
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correctly updated according to copy doc

## Issue / Card
[WD-8434](https://warthogs.atlassian.net/browse/WD-8434)

Fixes #

## Screenshots

[WD-8434]: https://warthogs.atlassian.net/browse/WD-8434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Sophie-32** [comment]
Screen Shot 2024-02-07 at 12 10 15 PM

I might be doing the demo wrong, but this form is on the blog page. Per the copydoc, the layout should be similar to the original. The example from the download page is just to show a sample of a form with only an email submission and checkbox.

**Comment by Sophie-32** [comment]
The checkbox should be required and the CTA button is not correct

**Comment by lizzochek** [comment]
@Sophie-32 accidentally touched another form. updated

**Comment by Sophie-32** [comment]
1. change placement of asterisk for checkbox
2. can you check the color of the text input box? It seems a little lighter than usual

**Comment by lizzochek** [comment]
@Sophie-32 1 and 3 updated. 2 - colour is the same as usual, I did not change anything about it

File: templates/blog/newsletter-form.html [hunk 1]
```diff
@@ -5,101 +5,46 @@ <h5 class="p-muted-heading u-no-margin--bottom">Newsletter signup</h5>
 
   <div class="blog-p-card__content">
     <form action="/marketo/submit" method="post" id="mktoForm_1212" onsubmit="ga('send', 'Newsletter', 'Signup', 'New homepage newsletter signup');">
```
Comment (mtruj013) [inline templates/blog/newsletter-form.html:7]:
```suggestion
    <form action="/marketo/submit" method="post" id="mktoForm_4960" onsubmit="ga('send', 'Newsletter', 'Signup', 'New homepage newsletter signup');">
```

File: templates/blog/newsletter-form.html [hunk 2]
```diff
@@ -5,101 +5,46 @@ <h5 class="p-muted-heading u-no-margin--bottom">Newsletter signup</h5>
...
-        <input type="hidden" aria-hidden="true" aria-label="hidden field" name="utm_source" id="utm_source" value="" />
-        <input type="hidden" aria-hidden="true" aria-label="hidden field" name="utm_content" id="utm_content" value="" />
-        <input type="hidden" aria-hidden="true" aria-label="hidden field" name="utm_term" id="utm_term" value="" />
-        <input type="hidden" aria-hidden="true" aria-label="hidden field" name="GCLID__c" id="GCLID__c" value="" />
-        <input type="hidden" aria-hidden="true" aria-label="hidden field" name="Facebook_Click_ID__c" id="Facebook_Click_ID__c" value="" />
-        <input type="hidden" aria-hidden="true" aria-label="hidden field" id="preferredLanguage" name="preferredLanguage" maxlength="255" value="" />
-        <input type="hidden" name="thankyoumessage" value="Thank you for subscribing!<br />You will begin receiving emails as new content is posted. You may unsubscribe any time by clicking the link in the email.">
-      </div>
+    
+          <input value="1212" name="formid" type="hidden">
```
Comment (mtruj013) [inline templates/blog/newsletter-form.html:31]:
```suggestion
          <input value="4960" name="formid" type="hidden">
```

**Review by mtruj013 — APPROVED** [review]
LGTM just need to change the marketo id

**Pull request merged**